ddtrace/internal/injection.py

```diff
@@ -91,15 +108,11 @@ def _inject_hook(code: Bytecode, hook: HookType, lineno: int, arg: Any) -> None:
     if not locs:
         raise InvalidLine("Line %d does not exist or is either blank or a comment" % lineno)
 
-    # DEV: This is the bytecode equivalent of
-    # >>> hook(arg)
-    # Additionally, we must discard the return value (top of the stack) to
-    # restore the stack to the state prior to the call.
     for i in locs:
         code[i:i] = INJECTION_ASSEMBLY.bind(dict(hook=hook, arg=arg), lineno=lineno)
 
 
-_INJECT_HOOK_OPCODE_POS = 0 if PY < (3, 11) else 1
+_INJECT_HOOK_OPCODE_POS = 0 if PY < (3, 11) or PY >= (3, 13) else 1
```

Comment:
So this is exclusive to 3.11 and 3.12?

Reply:
That's right, the opcodes changed a bunch in the last few minor versions